tests/test_utils.py

```diff
@@ -0,0 +1,21 @@
+class MockResponse:
+    def __init__(self, json_content):
+        self.json_content = json_content
+
+    def json(self):
+        return self.json_content
+
+
+def mock_get_factory(target_url, json_content):
+    """generate a mock to patch request.get with a json response"""
+
+    def mock_get(url, **kwargs):
+        assert url == target_url
+        assert kwargs == {
+            "fields": "product_name,code,last_editor,last_edit_dates_tags",
+            "sort_by": "last_modified_t",
+            "labels_tags_en": "vegan",
```

Comment:
@alexgarel can you please check it , I'm little lost in this

Reply:
Your kwargs will change from test to test. So you don't want to check here a static thing.

You have to do as for url, get the expected kwargs at the moment you create the mock, and then check against that.

See my suggestion below.